docs/cloud/datastores.md

```diff
@@ -86,6 +87,79 @@ By default a dragonfly cloud data store support redis cluster protocol and clien
 
 Multi node cluster is in private beta, please [schedule a meeting with a product expert](https://calendly.com/d/ymz-yhv-q8f/dragonfly-cloud?month=2024-07) to get access. 
 
+## Connecting to Data Store
+
+Once a data store *Status* is *Active* you can connect to it with any redis client using the *connection string* provided in the data store drawer. 
```

Comment:
nit: "Redis client"

Reply:
Is it worth an example of what a "connection string" looks like? (I hadn't heard of the term before, also maybe "connection URL" is clearer?)

Reply:
> nit: "Redis client"
Thanks, fixed.


> Is it worth an example of what a "connection string" looks like? (I hadn't heard of the term before, also maybe "connection URL" is clearer?)
Ok, redis uses the term connection URI so modified accordingly.
(Also in FE https://github.com/dragonflydb/controlplane/pull/2631)